src/one_dragon_qt/overlay/overlay_window.py

```
@@ -206,28 +148,71 @@ def paintEvent(self, event: QPaintEvent) -> None:
 
             label = self._format_vision_label(item)
             if not label:
                 continue
 
             text_h = painter.fontMetrics().height() + 4
-            text_w = min(rect.width() + 18, painter.fontMetrics().horizontalAdvance(label) + 8)
-            text_y = max(0, rect.top() - text_h - 2)
-            text_rect = QRect(rect.left(), text_y, max(40, text_w), text_h)
-
-            painter.fillRect(text_rect, QColor(0, 0, 0, 170))
+            # 背景框宽度按完整文本宽度自适应，不裁剪模板名
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)